pkg-manager/core/src/install/extendInstallOptions.ts

```diff
@@ -91,6 +91,7 @@ export interface StrictInstallOptions {
   unsafePerm: boolean
   registries: Registries
   tag: string
+  cmdFullName?: string
```

Comment:
Don't pass the command name. The installation engine doesn't need to have direct knowledge about the CLI. Add a boolean called `updateIfNewer`.

Reply:
I have renamed it to `preventDowngrade`.